ha/setup/post_disruptive_upgrade.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+"""Module which performs post-upgrade routines for the Disruptive Upgrade feature."""
+
+import os
+from shutil import copystat
+from xml.etree import ElementTree
+from xml.etree.ElementTree import Element
+
+from deepdiff import DeepDiff
+from cortx.utils.log import Log
+from ha.core.error import UpgradeError
+from ha.execute import SimpleCommand
+from ha.const import (BACKUP_DEST_DIR_CONF, CONFIG_DIR, SOURCE_CONFIG_PATH,
+                        RA_LOG_DIR, CHECK_PCS_STANDBY_MODE, \
+                        CLUSTER_STANDBY_UNSTANDBY_TIMEOUT, PCS_CLUSTER_STANDBY, \
+                        PCS_CLUSTER_UNSTANDBY)
+from ha.setup.create_pacemaker_resources import create_all_resources
+import yaml
```

Comment:
Please move this above at line #19 (after import os)

Reply:
@Madhura-08 Lets not use any new python package. Please check if a package can be used from https://github.com/Seagate/cortx-re/blob/main/scripts/third-party-rpm/python-requirements.txt. If not, then we should write our own code.

Reply:
I think its not a standard package. Its a third party package. Hence not imported with standard imports according to pylint suggestion